pkg/metrics/common.go

```diff
@@ -17,15 +17,103 @@ limitations under the License.
 package metrics
 
 import (
+	"regexp"
 	"time"
 
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
+	// StartingTime is the starting time of ndm.
 	StartingTime time.Time
-	Uptime       = prometheus.NewGauge(prometheus.GaugeOpts{
-		Name: "Uptime",
-		Help: "Uptime of node disk manager",
+	// Uptime is the uptime of ndm.
+	Uptime = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "ndm_uptime_seconds",
```

Comment:
can you add Namespace here also.Set it as environment variable.It will be helpful in grouping various instances into same namespace in kubernetes.

Reply:
Namespace for uptime only or others prometheus variables also?